implementations/rust/ockam/ockam_multiaddr/Cargo.toml

```diff
@@ -25,12 +25,16 @@ description = "An implementation of multiformats.io/multiaddr"
 
 [features]
 default = ["std"]
-std = ["ockam_core/std", "unsigned-varint/std", "serde?/std"]
+std = ["ockam_core/std", "unsigned-varint/std", "serde?/std", "minicbor/std"]
 cbor = ["minicbor"]
 
 [dependencies]
-minicbor = { version = "0.24.1", default-features = false, features = ["alloc"], optional = true }
-once_cell = { version = "1.19.0", default-features = false, features = ["alloc"] }
+minicbor = { version = "0.24.1", default-features = false, features = [
```

Comment:
Can we please keep the previous formatting here?

Reply:
@PsychoPunkSage you can run this command to automatically apply the expected format:

```
# from the project root directory
dprint fmt --config ./tools/dprint/dprint.json
```

Reply:
Done